_artifacts/toc.yml

```diff
@@ -1514,8 +1514,8 @@ main-menu:
         path: sandboxed_containers/deploying-sandboxed-container-workloads.adoc
```

Comment:
@tshwartz Please revert your changes to _artifacts/toc.yml. That file is not used by our current process.

Reply:
@GroceryBoyJr I'm new to Red Hat and not sure what this means. Please contact me when you get online.